plugins/bkpr/recorder.c

```diff
@@ -1935,11 +1935,12 @@ char *maybe_update_onchain_fees(const tal_t *ctx, struct db *db,
 }
 
 void maybe_closeout_external_deposits(struct db *db,
-			              struct chain_event *ev)
+			              struct bitcoin_txid *txid,
```

Comment:
```suggestion
			              const struct bitcoin_txid *txid,
```

usually we use `const` here